sites-config/ci_config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "artifactStructure": {
+      "assets": [
+          {
+              "root": "test-site/build"
+          }
+      ],
+      "hbsTemplates": []
+  },
+  "dependencies": {
+      "installDepsCmd": "npm install",
+      "requiredFiles": [".nvmrc"]
```

Comment:
I guess .nvmrc is not included by default but package.json and package-lock are?

Reply:
This is what I found on slack:
"the specified required_files are copied into the docker image, and then the install_deps cmd is ran. This happens  before the entire repo is copied into the image. So every file which is required by the install_deps command needs to be specified as a required file."
I assume the package files would have to be default since it installed and built fine in Sites. But I will update to include it nontheless.